<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrategySelectionDialog</class>
 <widget class="QDialog" name="StrategySelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>984</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Strategy Selection Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../DTMC_Simulator/resources/icons/FS4DTMC.PNG</normaloff>../../../DTMC_Simulator/resources/icons/FS4DTMC.PNG</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white</string>
     </property>
     <widget class="QComboBox" name="cb_strategy">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>50</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>MCS - Random walk</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MCS - Limiting distribution</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MCS - Cesaro limit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MCS - Modified cesaro limit</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MCS - Matrix powering</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MCS - Direct approach</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MCS - Forward approach</string>
       </property>
      </item>
     </widget>
     <widget class="QTextEdit" name="te_steps">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;100&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_strategy">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Choose appropriate strategy:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_steps">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Simulation steps:</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_stStVectorPlot">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>540</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Plot the stationary state distribution</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_representationForms">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>260</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Selected representation forms:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_cuStStVectorPlot">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>610</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Plot the cumulative stationary state distribution</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_coCuStStVectorPlot">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>680</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Plot the complementary cumulative stationary state distribution</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_stStVector">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>460</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Visualization of the stationary state distribution</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_alphaRelaxation">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Alpha-relaxation parameter:</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="te_alphaRelaxation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>180</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_stStVectorPlotSeparated">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>590</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Separated visualization</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_cuStStVectorPlotSeparated">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>660</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Separated visualization</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_coCuStStVectorPlotSeparated">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>730</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Separated visualization</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_graph">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>300</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Visualize the graph of the Markov chain</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_markClosures">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>390</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Mark closures</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_calculate_period">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>420</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Calculate period</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_graphLayout">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>350</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Circular layout</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spring layout</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spectral layout</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Random layout</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="la_graphLayout">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Graph layout:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_evoStAv">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>780</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Plot evolution of the state average</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_randomWalk">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>730</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Plot random walk</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_evoSt">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>830</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Evolution of state probabilities</string>
      </property>
     </widget>
     <widget class="QLabel" name="la_startState">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>870</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start state:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_startState">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>880</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="la_directScheme">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calculation scheme:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_directScheme">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>229</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Gaussian scheme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Inverse iteration scheme</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="cb_transitionMatrix">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>730</y>
        <width>761</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Visualize transition matrix</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_saveSSD">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>510</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Save results in external directory</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_saveTM">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>792</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Save results in external directory</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_filePath_SSD">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>510</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>C:/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bu_filePath_SSD">
      <property name="geometry">
       <rect>
        <x>822</x>
        <y>508</y>
        <width>100</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>URL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_filePath_TM">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>792</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>C:/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bu_filePath_TM">
      <property name="geometry">
       <rect>
        <x>822</x>
        <y>790</y>
        <width>100</width>
        <height>33</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>URL</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_period">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Period of the Markov chain:</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="te_period">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>135</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bb_acceptStrategy">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bb_acceptStrategy</sender>
   <signal>accepted()</signal>
   <receiver>StrategySelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_acceptStrategy</sender>
   <signal>rejected()</signal>
   <receiver>StrategySelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
